### ISSUE TITLE
JSON files not generted

### ISSUE THREAD

**Issue by icebruce** [kometenstaub/linked-data-helper#9]
**Steps to reproduce**
1. Install Linked Data vocabularies
2. Install Linked Data helper
3. Download LCSH -> https://id.loc.gov/download/ -> latest file "LC Subject Headings (LCSH) *NEW Pilot* (MADS/RDF and SKOS/RDF)" in .ndjson
4. Unzip
5. Set the path in Linked Data helper to the unzipped folder-> C:\Users\User\Downloads\LCSH
6. Leave "Folder for JSON files" empty
7. Start conversion
8. Message displays that conversion started

**Expected Results**
Files to be generated or maybe a message if/when it finished processing

**Actual results**
Nothing happens. Files not generated and no other notification. Tried with different file location of the unzipped LCSH. Maybe I'm doing something wrong?

**Comment by kometenstaub** [comment]
> Set the path in Linked Data helper to the unzipped folder-> C:\Users\User\Downloads\LCSH

The path needs to be set to the file, not the folder.

**Comment by icebruce** [comment]
>The path needs to be set to the file, not the folder.

Thanks! That fixed it. 

It did create a folder tilted "linked-data-vocabularies" under my attachments folder. I also got the confirmation in Obsidian that the json files have been created. Unfortunately when I open the new folder its empty..

Tried to extract with different folder locations and same result. The "conversion" took about 30 seconds, and I did see obsidian take up some memory during that time, so it looks like it kinda did something but not quite there

**Comment by kometenstaub** [comment]
Did you open it in Obsidian or with your OS file explorer? By default, these files will not show up in Obsidian's file explorer because they are JSON files, unless you activate show all files.

**Comment by icebruce** [comment]
It was with my OS (Win10), I have hidden files enabled and the folder is empty. I tested on two machine, a laptop and desktop, both on windows. When running the plugin, pointed to the new folder it says "JSON could not be read"

**Comment by kometenstaub** [comment]
Are you using the latest plugin version and seeing any errors in the console when you start the parsing?

**Comment by kometenstaub** [comment]
And what is your Obsidian installer version? Please make sure it is up-to-date.

**Comment by icebruce** [comment]
So I did some more testing and it somehow worked out finally.
My installer is 0.14.5 and the plugin is up to date. I deleted the folder and ran the conversion again which created the empty folder (no errors in the console). Then with the folder open in my explorer I ran the conversion once more and voila, the 3 files got populated in the folder...weird.

Anyways its working now so I will be playing with this. Thanks for the help!
